### PR TITLE
chore: Cleanup CI/CD and top level README

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

- Remove the sample code from top level README.md
- Add some description on how to add the package locally using melos
- Remove the unused release github actions. - Currently exploring [Melos action](https://github.com/marketplace/actions/melos-action) to see if we can truly automate the release process. 
- Add a Github action to check if PR title has the correct format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,605 +1,19 @@
...
+## Run locally
 
-</details>
+This repo is a monorepo powered by [Melos](https://melos.invertase.dev/) containing [supabase_flutter](https://github.com/supabase/supabase-flutter/tree/main/packages/supabase_flutter) and its sub-libraries. All packages are located in the `packages` directory.
 
----
+To install on a locally developed app:
+- Clone this repo
+- Run `melos bootstrap` or `melos bs` at the root of the cloned directory to install dependencies
+- Add the target package to your `pubspec.yaml` file specifying the path.
```
Comment (Vinzent03) [inline README.md:16]: What about adding a pubspec excerpt example here
```yml
supabase_flutter:
  path: <your-path-to-the-supabase_flutter-directory>
```

File: README.md [hunk 2]
```diff
@@ -1,605 +1,19 @@
...
-</plist>
-```
+## Run locally
 
-</details>
+This repo is a monorepo powered by [Melos](https://melos.invertase.dev/) containing [supabase_flutter](https://github.com/supabase/supabase-flutter/tree/main/packages/supabase_flutter) and its sub-libraries. All packages are located in the `packages` directory.
 
----
+To install on a locally developed app:
+- Clone this repo
```
Comment (Vinzent03) [inline README.md:14]: I think we can add the simple command to install melos here as well:
`dart pub global activate melos`

**Review by Vinzent03 — CHANGES_REQUESTED** [review]
So to understand the current publish process, the release prs only increase the version and add changelog, you pull and manually release each package?

**Comment by dshukertjr** [comment]
>So to understand the current publish process, the release prs only increase the version and add changelog, you pull and manually release each package?

Yes, I do publish the packages manually using Melos currently. 
Currently it looks like this:

1. Create `chore/publish` branch from `main`
2. Run `melos version`, which bumps the package version number, adds changelogs, updates version.dart files, and creates a Git tag.
3. Create a PR. 
4. After the PR is merged, pull `main` and run `melos publish`, which publishes all the updated packages.

**Review by Vinzent03 — APPROVED** [review]

**Pull request merged**